foundry.toml

```diff
@@ -1,7 +1,7 @@
 [profile.default]
   optimizer = true
   optimizer_runs = 10_000_000
-  solc_version = "0.8.16"
+  solc_version = "0.8.20"
```

Comment:
Are we assuming by the time this is released L2's would have implemented `PUSH0`? Or is this a non-issue

Reply:
See https://github.com/ScopeLift/foundry-template/pull/20#discussion_r1256017884, but pinned EVM version in e0c9e78f